src/app/components/shell/header/header.scss

```diff
@@ -74,606 +43,394 @@
+                transition-delay: 0.20s;
+            }
 
-                @at-root .sticky#{&} {
-                    display: block;
-                    order: 3;
-                    margin-left: 2rem;
-                }
+            &:nth-child(5) {
+                transition-delay: 0.25s;
             }
 
-            &:hover,
-            &:focus,
-            &:active {
-                outline: none;
+            a {
+                color: color(neutral-lightest);
+                display: inline-block;
+                font-family: 'Lato', Arial, Helvetica, sans-serif;
+                font-size: 1.6rem;
+                margin: 1.25rem 0 0.5rem -1rem;
+                padding: 0.5rem 2rem;
             }
+        }
+    }
 
-            span {
-                //create middle line
-                @include line;
-                position: relative;
-                transition: background 0.3s 0.2s ease;
-
-                /*create the upper and lower lines as pseudo-elements of the middle line*/
-                &::before,
-                &::after {
-                    @include line;
-                    position: absolute;
-                    left: 0;
-                    content: '';
-                    //set transform origin back to center
-                    transform-origin: 50% 50%;
-                    transition: top 0.3s 0.2s ease, transform 0.3s ease;
-                }
+    .expanded {
+        .sub-menu {
+            height: 21rem;
+            opacity: 1;
+            width: 30rem;
+        }
 
-                &::before { top: $button-size / 4; }
-                &::after { top: -$button-size / 4; }
+        span:nth-child(1),
+        span:nth-child(3) {
+            transform: rotate(45deg);
+            z-index: 100;
+        }
 
-                @at-root .active#{&} {
-                    transition: background 0.3s 0s ease;
-                    background: transparent;
+        span:nth-child(1) {
+            transform: rotate(45deg) translateY(0.2rem) translateX(0.8rem);
+        }
 
-                    &::before,
-                    &::after {
-                        transition: top 0.3s ease, transform 0.3s 0.2s ease;
-                        top: 0;
-                        width: $button-size;
-                    }
+        span:nth-child(2) {
+            height: 21rem;
+            overflow: visible;
+            right: -16rem;
+            top: 7rem;
+            width: 30rem;
+        }
 
-                    &::before {
-                        transform: rotate3d(0, 0, 1, 45deg);
-                    }
+        span:nth-child(3) {
+            transform: rotate(-45deg) translateY(-0.7rem) translateX(1.1rem);
+        }
 
-                    &::after {
-                        transform: rotate3d(0, 0, 1, -45deg);
-                    }
-                }
-            }
-        } //end expand-nav button
-    } //main-nav
+        li {
+            margin-right: 12rem;
+        }
+    }
 }
```

Comment:
Im not sure why you are rewriting this scss for the header? it seems what you are rewriting has nothing to do with the home page functionality. This is all relative to the header alone and I am responsible for this code.